didpass-demo/verifier-demo/.env.example

```diff
@@ -0,0 +1,10 @@
+NEXT_PUBLIC_URL=http://localhost:3002
+
+VERIFIER_PRIVATE_KEY_PEM="-----BEGIN PRIVATE KEY-----\nMIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgln38K+VhuqmDCahN\nVFfftgyM7MeY7t6LZqIx+PcPCjOhRANCAASvtotehwDpyUwONiDzQDxWdwaa+uajtzi4SPgPtJm5YGsLINywg+uh9MaHTRmVLlgCgIGenyT0Yx6iNmPRo3zb\n-----END PRIVATE KEY-----\n";
```

Comment:
@Adrian-py ini ada `;` di end of line bikin error, file env gak perlu ada `;` ya dibelakangnya